python/cudf/cudf/io/parquet.py

```diff
@@ -320,10 +326,15 @@ def to_parquet(
         if index is None:
             index = True
 
+        # Convert partition_file_name to a call back
+        if partition_file_name:
+            partition_file_name = lambda x: partition_file_name
```

Comment:
To silence `flake8` warning about assigning `lambda`s.

```suggestion
            partition_file_name = lambda x: partition_file_name  # noqa: E731
```